atm.js

```diff
@@ -1,21 +1,56 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-}
-
-function checkingWithdrawal(amount){
-
-}
-
-function savingsDeposit(amount){
-
-}
-
-function savingsWithdrawal(amount){
-
-}
-
-// an eventListerner for each button, each one a "click"
\ No newline at end of file
+$(document).ready(function(){
+
+  // deposit money using deposit button
+
+  $("#depositCheckingButton").on("click", function(){  //Added an event listener to Deposit Checking Balance
+    console.log("I clicked deposit"); // Making sure my event has fired
+    var checkingInput = parseInt($("#checkingInput").val()) //Setting up a variable for a Deposit
```

Comment:
i really liked the messages you put in your console.log, so you really know whats going on as you log things.